### DISCUSSION TITLE
Applying hank in open source repos

### DISCUSSION THREAD

**Discussion by pbrudnick** [AdRoll/rebar3_hank#97] [General]
https://github.com/erlef/rebar3_hex/pull/220

**Comment by elbrujohalcon** [comment]
Great idea!!

I can recommend a couple of "victims" :trollface: …

- All the Erlang repos under http://github.com/inaka
- https://github.com/WhatsApp/erlfmt/
- https://github.com/erlang/rebar3

**Comment by juanbono** [comment]
Some ideas:
* https://github.com/elli-lib/elli 
* https://github.com/talentdeficit/jsx
* https://github.com/erlcloud/eini
* https://github.com/erlcloud/erlcloud
